internal/service/vpc/resource_vpc_peering_connection.go

```diff
@@ -37,7 +37,7 @@ var aivenVPCPeeringConnectionSchema = map[string]*schema.Schema{
 	},
 	"peer_region": {
 		ForceNew: true,
-		Optional: true,
+		Required: true,
```

Comment:
Sorry, perhaps I wasn't clear enough about making `peer_region` required. This resource is deprecated and coming from v2. It combines all types of VPC peering connections in one, like AWS, Azure and GCP. Therefore we mustn't make `peer_region` as required.